patches/chrome-test-BUILD.gn.patch

```diff
@@ -10,3 +10,11 @@ index fbbea351bd8ad009c81a41a7d1ba0590611c21bb..edc96d7f0b52ae06b336ff6d32ca055d
      # Runtime dependencies
      data_deps = [
        "//chrome:browser_tests_pak",
+@@ -3699,6 +3700,7 @@ test("unit_tests") {
+   deps = [
+     ":test_support",
+     ":test_support_unit",
++    "//brave/test:brave_test_support_unit",
```

Comment:
we don't need this patch as mentioned in DM, but patches should always be extensible so if we did patch, we would do something like `deps += brave_chrome_unit_tests_deps`

Reply:
I thought that "npm run test -- unit_tests -- Debug" built the Chromium unit_tests. But,  if unit_tests means brave_unit_tests or Brave doesn't maintain the unit_tests, this PR is not unnecessary. Thanks for reviewing.

Reply:
you should be running `brave_unit_tests` which includes `push_messaging_service_unittest.cc` in the sources

Reply:
Ok.